src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/createbundle/CreateBundleAboutToSubmitHandler.java

```diff
@@ -157,14 +157,14 @@ private void setBundleConfig(SscsCaseData sscsCaseData) {
 
     private void setMultiBundleConfig(SscsCaseData sscsCaseData, PreSubmitCallbackResponse<SscsCaseData> response) {
         List<MultiBundleConfig> configs = new ArrayList<>();
-
+        
         if (sscsCaseData.getDwpDocuments().stream().filter(f -> (f.getValue().getDocumentType().equals(DwpDocumentType.DWP_RESPONSE.getValue())
                 && f.getValue().getEditedDocumentLink() != null)
                 || f.getValue().getDocumentType().equals(DwpDocumentType.DWP_EVIDENCE_BUNDLE.getValue())
-                && f.getValue().getEditedDocumentLink() != null).count() > 0) {
+                && f.getValue().getEditedDocumentLink() != null).count() > 0 &&  checkPhmeReviewIsGrantedOrUnderReview(sscsCaseData))  {
```

Comment:
The brackets here don'y look right. Looks like you are only hitting the 'checkPhmeReviewIsGrantedOrUnderReview' method within the DWP_EVIDENCE_BUNDLE block, when I think it should be for everything. Should probably add a test for this specific scenario which I think will break.

Reply:
ignore me